wai-aria/tools/atta-example.py

```diff
@@ -0,0 +1,289 @@
+# atta-example
```

Comment:
It does work for me.

Related: The files/cors.headers and files/aria.headers are, I believe, only needed by the examples and could then be removed from what is to be committed.